packages/web/docs/src/pages/docs/features/app-deployments.mdx

```diff
@@ -339,6 +343,102 @@ curl \
   http://localhost:4000/graphql
 ```
 
+</Tabs.Tab>
+
+<Tabs.Tab>
+
+The code demonstrates how to send a persisted document request using Apollo Client. Make sure to
+replace the placeholders with your actual values.
+
+```typescript filename="Apollo Client Persisted Document Configuration"
+import { ApolloClient, HttpLink, InMemoryCache } from '@apollo/client'
+import { createPersistedQueryLink } from '@apollo/client/link/persisted-queries'
+
+// replace <app_name>, <app_version>, and with your values
+const appVersionPrefix = '<app_name>~<app_version>~'
+
+const persistedQueriesLink = createPersistedQueryLink({
+  generateHash(document) {
+    // GraphQL Code Generator Client Preset generates
+    // the persisted document ID within the document node.
+    // For other implementations return the hash prefixed with the app version.
+    return appVersionPrefix + document['__metadata__']['hash']
+  }
```

Comment:
wondering we should do this incase someone doesn't have hashes generated with codegen
```ts
if (!document?.['__metadata__']?.['hash'] {
  throw Error("Hash not available for document")
}
```